src/main/java/org/eclipse/basyx/extensions/internal/storage/BaSyxStorageAPI.java

```diff
@@ -56,6 +60,11 @@ public abstract class BaSyxStorageAPI<T> implements IBaSyxStorageAPI<T> {
 	protected final String COLLECTION_NAME;
 	protected final Class<T> TYPE;
 
+	BaSyxStorageAPI() {
```

Comment:
public?

Reply:
The visibility is set to package visibility on purpose as this constructor is only used to be able to mock this abstract class with Mockito.